src/python/pants/option/global_options.py

```diff
@@ -435,7 +435,18 @@ def register_bootstrap_options(cls, register):
             help="Paths to ignore for all filesystem operations performed by pants "
             "(e.g. BUILD file scanning, glob matching, etc). "
             "Patterns use the gitignore syntax (https://git-scm.com/docs/gitignore). "
-            "The `--pants-distdir` and `--pants-workdir` locations are inherently ignored.",
+            "The `--pants-distdir` and `--pants-workdir` locations are inherently ignored."
+            "--pants-ignore can be used in tandem with --pants-use-gitignore, and any rules "
+            "specified here apply after rules specified in a .gitignore file.",
+        )
+        register(
+            "--pants-use-gitignore",
```

Comment:
Bump that I recommend `--pants-ignore-use-gitignore` for a consistent namespace with `--pants-ignore`. It's verbose, but this is an advanced option that will almost never actually be changed, and when it is, only by one Pants admin.

I think the namespace consistency is worth the verbosity.